src/OpenLoco/src/Map/MapGenerator/HeightMap.cpp

```diff
@@ -0,0 +1,32 @@
+#include "HeightMap.h"
+#include <algorithm>
+
+namespace OpenLoco::World::MapGenerator
+{
+    constexpr uint8_t kHeightmapMarkedFlag = (1 << 7);
+
+    void HeightMap::resetMarkerFlags()
+    {
+        std::for_each_n(data(), size(), [](uint8_t& value) { value &= ~kHeightmapMarkedFlag; });
+    }
+
+    uint8_t HeightMap::getHeight(Point pos) const
```

Comment:
I think this should all be refactored into using TilePos types. Filed #2386 to discuss.